### PR TITLE
First draft of 11-12 release notes

### PR DESCRIPTION
**Pull request by srpearce**
@mhoeber @lamagnifica @catong Please review.

I'm thinking about adding https://github.com/edx/edx-platform/pull/5835; your thoughts?

### CODE REVIEW

File: en_us/release_notes/source/11-12-2014.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+  Recent changes include:
+
+  * The `Add a Content Experiment in OLX`_ section now informs users that
+    content experiments can only be defined at the unit level. (DOC-1058 and
+    DOC-1161)
+
+  * The `Show Answer`_ information has been updated with reset information. (DOC-1146)
+
+.. _Add a Content Experiment in OLX
```
Comment (mhoeber) [inline en_us/release_notes/source/11-12-2014.rst]: not sure why this line is there; looks like bullet for OLX guide was lost.  there's one change log entry to include for it

File: en_us/release_notes/source/11-12-2014.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+  * Student documentation now includes an `edX Video Player`_ section.
+
+.. adding line for space
+
+* `Creating Peer Assessments`_
+
+* `edX Research Guide`_
+  
+* `edX Platform Developer's Guide`_
```
Comment (mhoeber) [inline en_us/release_notes/source/11-12-2014.rst:87]: one change log entry in developers guide to include

**Comment by mhoeber** [comment]
adding @ormsbee as release master

File: en_us/release_notes/source/11-12-2014.rst (hunk 1 above)
Comment (srpearce) [inline en_us/release_notes/source/11-12-2014.rst]: Do you mean the one at line 52?

File: en_us/release_notes/source/11-12-2014.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+You can also sign up for the `edX Release Announcements mailing list`_ to get
+an email message when we update the edX Course Staff Release Notes.
+
+
+*************
+edX Studio
+*************
+
+* The **Course Export** page now includes a warning that exported data includes
+  information such as MATLAB API keys and LTI passports. (DOC-1138)
```
Comment (lamagnifica) [inline en_us/release_notes/source/11-12-2014.rst]: sensitive information (?)

File: en_us/release_notes/source/11-12-2014.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    that content experiments can only be defined at the unit level. (DOC-1058
+    and DOC-1161)
+
+  * Added reset information to the `Show Answer`_ section. (DOC-1146)
+
+.. adding line for space
+
+
+* `Using edX Insights`_
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/11-12-2014.rst:65]: Click **Help** in the upper-right corner of any page in the edX Insights user interface to access relevant sections of this guide. You can also download the entire guide as a PDF.

**Comment by lamagnifica** [comment]
I like the idea of adding 5835

File: en_us/release_notes/source/11-12-2014.rst [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+.. adding line for space    
+    
+* `EdX Guide for Students`_
+
+  Recent changes include:
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/11-12-2014.rst:75]: your PR 15, the scientific notation + calculator chapter?

File: en_us/release_notes/source/11-12-2014.rst [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+  download the entire guide as a PDF.
+
+  Recent changes include:
+
+  * Added a cautionary note to `Add a Content Experiment in OLX`_ explaining
+    that content experiments can only be defined at the unit level. (DOC-1058
+    and DOC-1161)
+
+  * Added reset information to the `Show Answer`_ section. (DOC-1146)
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/11-12-2014.rst:60]: - that the scientific notation chapter is now in the SFD?

**Pull request merged**